app/helpers/application_helper.rb

```diff
@@ -1,2 +1,11 @@
 module ApplicationHelper
+  # Returns the full title on a per-page basis.
+  def full_title page_title = ""
+    base_title = I18n.t :base_title
```

Comment:
I18n

Reply:
Dòng này có vấn đề gì ạ??

Reply:
em chỉ cần viết thế này thôi:
```suggestion
    base_title = t :base_title
```